packages/typespec-ts/test/unit/modelsGenerator.spec.ts

```diff
@@ -236,9 +239,12 @@ describe("Input/output model type", () => {
     it("should handle enum -> string_literals", async () => {
       const cadlTypeDefinition = `
       #suppress "@azure-tools/typespec-azure-core/use-extensible-enum" "for test"
+      #suppress "@azure-tools/typespec-azure-core/documentation-required" "for test"
```

Comment:
do we still need this now that we added @doc

Reply:
Yes because the relevant enum type requres a doc